New definitions to add (#33)
datumforge/docs: product/Glossary.mdx

---

## A
**Account**

An Account is a billable customer group. Accounts may be aggregated into "Parent Accounts" and subdivided into "Projects". 

## B
**B2B**

Business to business. These businesses sell goods or services to other businesses, rather than to individual users. Datum's platform is optimized for B2B SaaS business models. 

**B2C**

> Business to customer. These businesses sell goods or services to individual customers, rather than to other businesses. 

Reviewer: ```suggestion
Business to consumer. These businesses sell goods or services to individual customers, rather than to other businesses. 
```

## C
**Compliance**

The act of implementing, monitoring, and providing evidence for controls that meet established guidelines such as SOC2, GDPR, or ISO 27001. 

**Contacts**

> Contacts are those people or entities you intend to target with marketing campaigns such as emails or other notifications. Contacts may be a simple email address or may include more infomration, such as name or phone number. Contacts may be organized into Private or Public lists for managing anything from marketing emails to system messages. 

Reviewer: ```suggestion
Contacts are those people or entities you intend to target with marketing campaigns such as emails or other notifications. Contacts may be a simple email address or may include more information, such as name or phone number. Contacts may be organized into Private or Public lists for managing anything from marketing emails to system messages. 
```

## D 

## E
**End User Customer**

> End User Customers are your customers. As a Tenant of Datum, you will log into Datum systems to execute activites that help you better serve your End User Customers. While Datum will help you manage entitlements and features for your End User Customers, they will only use your systems; not Datum directly. 

Reviewer: ```suggestion
End User Customers (EUC) are your customers. As a Tenant of Datum, you will log into Datum systems to execute activities that help you better serve your End User Customers. While Datum will help you manage entitlements and features for your End User Customers, they will only use your systems; not Datum directly. 
```

**Entitlement**

**Event**

Events are any activity a user executes, such as clicking buttons, making changes, or viewing pages. Events are ingested and can be reported on by Datum.

## F

**Feature**

> A feature performs a specific function or capability and provides value to users. They can be basic or complex, and are designed to meet specific requirements and objectives. Features can be made availabel through the use of Entitlements or Feature Flags.

Reviewer: It looks like you lost some of the changes from your last PR 

```suggestion
A feature performs a specific function or capability and provides value to users. They can be basic or complex, and are designed to meet specific requirements and objectives. Features can be made available through the use of Entitlements or Feature Flags.
```

## L

## M
**Member** 

Members are people you have invited to manage your workspace at Datum. These are the builders and operators who will log into the Datum portal to configure your workspace and view your customers, contracts, vendors, and more. 

## N

**NDA**

> A Non-Disclosure Agreement is a *must have* for Tenants and any of their software Vendors. NDAs are also frequenly put into place between a Tenant and a Customer or potential Customer. The NDA protects your proprietary information, so we at Datum offer you a template and tracking to ensure you don't miss creating this key artifact. 

Reviewer: ```suggestion
A Non-Disclosure Agreement is a *must have* for Tenants and any of their software Vendors. NDAs are also frequently put into place between a Tenant and a Customer or potential Customer. The NDA protects your proprietary information, so we at Datum offer you a template and tracking to ensure you don't miss creating this key artifact. 
```

## O
**Open Source** 

Open source software is software with source code that anyone can inspect, modify, and enhance. Datum is an example of an [open-source company] (https://github.com/datumforge).

**Organization**

Organizations are the highest level grouping within Datum. Organizations are made up of users and represent a billable unit. 

## P

**Project**

A Project is a grouping inside an account. Typically, users leverage projects to separate development and production environments or otherwise group assets for shared identification and management. 

**Parent Account**

> A Parent Account is a grouping of Accounts for the purposes of unifying the customer experience (this could be for the purpose of billing, entitlments, or shared contacts). 

Reviewer: ```suggestion
A Parent Account is a grouping of Accounts for the purposes of unifying the customer experience (this could be for the purpose of billing, entitlements, or shared contacts). 
```

## T

**Tenant** 

A Tenant at Datum is the business using Datum's software and tools to build, operate, and scale a cloud-based business!

## U

**User**

> User and "End User Customer" are used interchangeably at Datum. Users are your customers. As a Tenant of Datum, you will log into Datum systems to execute activites that help you better serve your End User Customers. While Datum will help you manage entitlements and features for your End User Customers, they will only use your systems; not Datum directly. 

Reviewer: ```suggestion
User and "End User Customer" are used interchangeably at Datum. Users are your customers. As a Tenant of Datum, you will log into Datum systems to execute activities that help you better serve your End User Customers. While Datum will help you manage entitlements and features for your End User Customers, they will only use your systems; not Datum directly. 
```
